lib/Partitioner/Partitioner.cpp

```diff
@@ -456,7 +456,7 @@ void Partitioner::doPartitioning(Function *F, NodeToFunctionMap &mapping) {
   // For any dependency that crosses a partition, add a placeholder and save
   // node. Record the dependence in the function graph.
   int logicalID = 0;
-  llvm::DenseMap<Node *, Placeholder *> placeholders;
+  std::unordered_map<NodeValue, Placeholder *> placeholders;
```

Comment:
Curious why the switch to `unordered_map` instead of `DenseMap`

Reply:
llvm::DenseMap doesn't allow me to use NodeValue as a key.  The error message is like error: no member named 'getEmptyKey' in 'llvm::DenseMapInfo<glow::NodeValue>'. Is still looking for a proper explanation to this error message.